cpp/log/strict_consistent_store.h

```diff
@@ -29,8 +29,8 @@ class StrictConsistentStore : public ConsistentStore<Logged> {
 
   util::Status SetServingSTH(const ct::SignedTreeHead& new_sth) override;
 
-  util::Status AssignSequenceNumber(const int64_t sequence_number,
-                                    EntryHandle<Logged>* entry) override;
+  util::Status UpdateSequenceMapping(
+      EntryHandle<ct::SequenceMapping>* entry) override;
```

Comment:
Adding the ```override```s in this file. Could also have remove the signing of entries in etcd in a separate commit, but after all the rest, that was easy. :wink: